pubsub/src/manager.rs

```diff
@@ -0,0 +1,88 @@
+//! Provides an executor for subscription Futures.
```

Comment:
I think we should expand on this module description a bit. The subscription manager is opinionated way to handle processing subscriptions that are based on `Stream` coming from the user codebase.
The manager takes care of:
1. Assigning the ID
2. Consuming a `Stream` of events (coming from user code) and transforming it into a subscription notifications to the client.
3. Spawning a resulting future to executor
4. Cancelling the stream when the subscription is canceled.